src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -324,6 +324,9 @@ class Frustum {
   double sphericity_{std::numeric_limits<double>::signaling_NaN()};
   double radius_{std::numeric_limits<double>::signaling_NaN()};
   double phi_{std::numeric_limits<double>::signaling_NaN()};
+  double half_opening_angle_{std::numeric_limits<double>::signaling_NaN()};
+  double one_over_tan_half_opening_angle_{
+      std::numeric_limits<double>::signaling_NaN()};
 };
 
 bool operator!=(const Frustum& lhs, const Frustum& rhs);
```

Comment:
[Random line] Should there be a test for the changes to the Frustum?

Reply:
I will add a random `opening_angle` to the test suite for map to check that the map is valid for various opening angles with the correct inverse/jacobian. Additional tests for "does this map do what we want" will be taken care of by the BCO domain's "check connectivity" test that checks that the Frustums with opening angles conform to the Wedges with opening angles.